examples/src/main/scala/sttp/client3/examples/GetAndParseJsonZioCirce.scala

```diff
@@ -1,11 +1,10 @@
 package sttp.client3.examples
 
+import io.circe.generic.auto._
 import sttp.client3._
-import sttp.client3.circe._
 import sttp.client3.asynchttpclient.zio._
-import io.circe.generic.auto._
-import zio._
-import zio.Console
+import sttp.client3.circe._
+import zio.{Console, _}
```

Comment:
You can clean that import

Reply:
No in fact as `Console` is used  in lines 21 and 22

Reply:
true but it is already imported by `import zio._`

Reply:
okk, curios why InteliJ did't remove it